source/Format.cpp

```diff
@@ -101,6 +105,7 @@ string Format::Number(double value)
 				result += ',';
 		}
 	}
+	while(rounded || right);
```

Comment:
For `do {} while` loops in Endless Sky code, the `while` should be on the same line as the closing `do` brace.

Reply:
Thanks.